### ISSUE TITLE
Problem with Zenbook 14 UX433FN

### ISSUE THREAD

**Issue by jhaotingc** [hieplpvip/ASUS-ZENBOOK-UX410-HACKINTOSH#75]
Hi, 

I've tried to install Catalina on my UX433FN, however there are still problems on type-c, HDMI, Audio and NullEthernet. 
Have anyone succeeded installing hackintosh on UX433FN?
This is my [EFI](https://github.com/jack34672/ZenBook14macOS).
btw thanks hieplpvip for his work on Zenbooks!

Thanks

**Comment by TheNewJavaman** [comment]
@jack34672 Did you ever get your Hackintosh working? I am on a UX433F with Whiskey Lake

**Comment by jhaotingc** [comment]
@TheNewJavaman I did get it working, but type-c, audio, hdmi port are not working. I can only access ethernet through usb ports.

**Comment by TheNewJavaman** [comment]
Thank you for the response! One other concern: did you get the default Intel WiFi card to work? I've been looking at [itlwm](https://github.com/OpenIntelWireless/itlwm)

**Comment by jhaotingc** [comment]
No I did not. If you get UX433F done can you share your EFI to me thanks!

**Comment by TheNewJavaman** [comment]
Can confirm that [AirportItlwm.kext](https://openintelwireless.github.io/itlwm/) allows native WiFi config and usability with the generic Intel chipset :)

Will upload EFI after I figure out keyboard issues

**Comment by jhaotingc** [comment]
Thanks a lot man!

**Comment by TheNewJavaman** [comment]
I uploaded my EFI files [here](https://github.com/TheNewJavaman/asus-zenbook-ux433fn-hackintosh-opencore-efi). I fixed Intel WiFi and default audio. Still working on display brightness, keyboard lighting, type-c, and hdmi. Please note that the included kexts are all debug versions. I will update them to release versions once I've fixed all the bugs.

**Comment by TheNewJavaman** [comment]
Oh, also: I had to remap alt/option to option/alt in the macOS preferences/keyboard/modifier keys settings pane

**Comment by jhaotingc** [comment]
Hey @TheNewJavaman I've tried your EFI and everything works perfectly except HDMI and keyboard lighting! Huge thanks!!
Type-c port and display brightness works fine on my UX433FN. Thanks again!

**Comment by TheNewJavaman** [comment]
Yeah no problem! I'm still working on camera, which I need for school, but I'm really happy that yours works too!

**Comment by jhaotingc** [comment]
@TheNewJavaman I have a question. My audio usually does not work (only works sometime).
I installed VoodooHDA (simply download it then kextload) and the controls pops out but there's still no sound.
How did you solve the audio problem? Thanks for your time!

**Comment by TheNewJavaman** [comment]
@jack34672 For me, the issue was dual booting with Windows. My Windows update Realtek drivers disabled macOS from initializing audio support with Apple alc. I have no idea why. To fix this issue, I opened device manager and "uninstalled device" for realtek sound controller.

**Comment by jhaotingc** [comment]
Thanks again man! That's exactly the problem. No wonder the sound is good when I boot from linux and failed from windows.

**Comment by ccnmxns** [comment]
> Yeah no problem! I'm still working on camera, which I need for school, but I'm really happy that yours works too!

Hey, have you fixed the Camera issue?

**Comment by TheNewJavaman** [comment]
@Suzamax not yet. It's turning out to be quite difficult. None of the kexts I've tried have been working.

**Comment by ccnmxns** [comment]
I have an AzureWave too. The only way I have to make the camera work is using a QEMU VM inside the Hackintosh and doing USB Passthrough.

I’m currently investigating this problem, I hope we’ll fix it :-)

Cheers!

**Comment by TheNewJavaman** [comment]
@Suzamax Would you mind dropping your UVC webcam specs? (vendor, id, etc)

**Comment by ccnmxns** [comment]
Yes! Sorry for being late.

Here's my webcam. The manufacturer is AzureWave.

USB2.0 HD UVC WebCam:

  N.º de modelo:	UVC Camera VendorID_5075 ProductID_22237
  Identificador único:	0x14400000*********

**Comment by ccnmxns** [comment]
Also: USB passthrough to a VMware Fusion .vmx VM with the PID and VID of the camera works!

Add this line onto the bottom of the .vmx:
```usb.autoConnect.device0 = "vid:CHANGEME pid:CHANGEME autoclean:0"```

Replace with your camera's Vendor ID and Product ID in hexadecimal.
